### ISSUE TITLE
Ability to remove language with `language_codes` is broken since 1.37.5

### ISSUE THREAD

**Issue by steffenbusch** [pglombardo/PasswordPusher#1904]
## 🐛 Bug Report

Since the `release` tag on dockerhub has recently moved forward from 1.36.5 to 1.37.7, my `settings.yml` configuration does no longer limit the available languages in the user-interface. Instead of just the two configured languages "de" and "en", now all languages are shown.

## 🔬 How To Reproduce

Excerpt of my `settings.yml`, which is mounted into the container as `/opt/PasswordPusher/config/settings.yml`

```yml
### Language & Internationalization
#
# List of supported languages indexed by language code.  This is used
# to build the in application language menu.
#
# To remove the availability of languages from the application entirely,
# comment out the language code(s) from the list below.
#
# <language code>: '<language name>'
language_codes:
  de: 'Deutsch'
  en: 'English'

# Used internally for the language selector model.
# This provides a conversion of language codes to
# country codes show the correct flag icons.
# See:
# https://github.com/lipis/flag-icons
# https://flagicons.lipis.dev
country_codes:
  de: :de
  en: :gb
```

I have started the container with various versions and it was working until (including) 1.37.4. 
With **1.37.5** my configuration was no longer respected and all languages are shown.

### Environment

Where are you running/using Password Pusher?

- [ ] pwpush.com
- [x] Docker Image
  - [x] pwpush
  - [ ] custom image
- [ ] Heroku
- [ ] Digital Ocean
- [ ] Microsoft Azure
- [ ] Google Cloud
- [ ] AWS
- [ ] Source Code
- [ ] Other (please specify)

If applicable, what version of Password Pusher?

### Screenshots

Starting with 1.37.5 it shows all languages despite the configuration to only show "de" and "en":

![image](https://github.com/pglombardo/PasswordPusher/assets/37350514/cca002d8-80b2-4134-8e1f-87d7d370551f)

## 📈 Expected behavior

In 1.36.5 and until including 1.37.4 it was like this:

![image](https://github.com/pglombardo/PasswordPusher/assets/37350514/7c8d3898-ca5b-4882-acc1-2f234e64422f)

**Comment by pglombardo** [comment]
Hi @steffenbusch - Apologies for the issue.  This is a side affect of #1875.

Let me give it some thought on the best way to fix this or an easy work-around...

**Comment by pglombardo** [comment]
Hey @steffenbusch - it took a few attempts to figure out a reliable work-around but I think I got one for you.

This worked previously for you because of a bug in the merging of `settings.yml` files.  Disabling languages is something I've considered but never directly added support for but...

If you put the file `config/initializers/xlimit_langs.rb` into your container with the following content, you should get your original setup back.

```
I18n.available_locales = [:en, :de]
```

An example command line might look like the following:

```sh
docker run -d \
      --mount type=bind,source=/path/settings.yml,target=/opt/PasswordPusher/config/settings.yml \
      --mount type=bind,source=/path/xlimit_langs.rb,target=/opt/PasswordPusher/config/initializers/xlimit_langs.rb \
      -p "5100:5100" pglombardo/pwpush:release
```

Could you create that file, give this a try and let me know?

**Comment by steffenbusch** [comment]
Hi @pglombardo Thank you very much for the workaround. This works perfectly for me. 🤩

Regarding:
> Disabling languages is something I've considered but never directly added support

Wouldn't it make sense to adjust this comment here in the `settings.yml`
https://github.com/pglombardo/PasswordPusher/blob/master/config/settings.yml#L829-L830

```yml
# To remove the availability of languages from the application entirely,
# comment out the language code(s) from the list below.
```

Or maybe refer to your workaround https://github.com/pglombardo/PasswordPusher/issues/1904#issuecomment-1957946876 there?

**Comment by pglombardo** [comment]
You're right/my bad.  I think that shows I'm juggling way too many things :-)

I'll figure out the best way to get that original functionality back eventually.

**Comment by steffenbusch** [comment]
Never mind. Your app is simply awesome and very impressive.

**Comment by LucasDouradoLinx** [comment]
Hi !!

When i comment all languages, in builded docker return error:

ERROR: failed to solve: process "/bin/sh -c bundle exec rails assets:precompile && bundle exec rake db:setup" did not complete successfully: exit code: 1

![image](https://github.com/pglombardo/PasswordPusher/assets/119966133/c1895792-d600-4b64-beda-3edd2afa0d73)

How block all languages? 

Is possible changed default to PT:BR?

I use docker in WebApp azure

**Comment by pglombardo** [comment]
Fix for this issue is coming in #1960.  It should be released some time this week.

**Comment by steffenbusch** [comment]
Thank you very much. I have followed the steps and the fix in v1.38.0 is working perfectly 👍
